ui/src/tempVars/components/TemplateVariableEditor.tsx

```diff
@@ -59,6 +60,7 @@ const TEMPLATE_BUILDERS = {
   [TemplateType.FieldKeys]: FieldKeysTemplateBuilder,
   [TemplateType.TagKeys]: TagKeysTemplateBuilder,
   [TemplateType.TagValues]: TagValuesTemplateBuilder,
+  [TemplateType.MetaQuery]: MetaQueryTemplateBuilder,
```

Comment:
How do you feel about moving this whole const to a constants file?

Reply:
^ Same comment about constants above.